### ISSUE TITLE
CommonJS bundle contains a syntax error

### ISSUE THREAD

**Issue by N2D4** [supabase-community/pg-gateway#23]
# Bug report

## Describe the bug

When I import pg-gateway with `require` (or my bundler transpiles my import calls into that), pg-gateway throws an import error.

## To Reproduce

Using the PGlite example from the [next branch](https://github.com/supabase-community/pg-gateway/blob/next/README.md), but with the imports at the top replaced by `require`, throws an error:

```js
# index.cjs
const { PGlite } = require('@electric-sql/pglite');
const { createServer } = require('node:net');
const { fromNodeSocket } = require('pg-gateway/node');
```

```sh
$ node index.cjs
Debugger listening on ws://127.0.0.1:64418/6c461da5-f0a8-4d36-8ef9-e05ec5c48608
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
/node_modules/.pnpm/pg-gateway@0.3.0-beta.3/node_modules/pg-gateway/dist/chunk-OSDXSGAD.cjs:1
"use strict";Object.defineProperty(exports, "__esModule", {value: true}); function _nullishCoalesce(lhs, rhsFn) { if (lhs != null) ...........................................................................................................................................................................................; exports.n = zA; exports.o = L; exports.p = w; exports.q = m; exports.r = KI; exports.s = l; exports.t = XA; exports.u = uI; exports.v = sA; exports.w = BA;
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            

SyntaxError: Unexpected token ','
    at internalCompileFunction (node:internal/vm:73:18)
    .........
```

## System information

- Version of pg-gateway: 0.3.0-beta-3
- Version of Node.js: 20.11.1

**Comment by gregnr** [comment]
Thanks for reporting @N2D4. Looks like a bug with tsup: https://github.com/egoist/tsup/issues/1124. Going to dig into this to see what our options are.

**Comment by gregnr** [comment]
@N2D4 this should be fixed now in `0.3.0-beta.4` via https://github.com/supabase-community/pg-gateway/pull/24. Would you mind giving this another try?
